test/pageobjects/login-page.js

```diff
@@ -0,0 +1,42 @@
+const { $ } = require("@wdio/globals");
+const Page = require("./page");
+
+class LoginPage extends Page {
+  open() {
+    return super.open("customer/account/login/");
+  }
+
+  get inputEmail() {
+    return $("//input[@id='email']");
+  }
```

Comment:
Did you tried to use id as selector for those input fields 
```
    get inputEmail() {
        return $('#email');
    }
```
For cases where you can use an ID as the selector, it’s a better option because it’s faster and more readable. When an ID cannot be used, it’s okay to use XPath.

Reply:
I tried using the ID selector ($('#email')), but I’m getting an error: Error: Can't call setValue on element with selector "#email" because element wasn't found. It seems the ID selector can’t locate the element during tests.